example_test.go

```diff
@@ -480,3 +480,66 @@ func ExampleCountersignature() {
 	// signature countersignature verified
 	// verification error as expected
 }
+
+// This example demonstrates signing and verifying COSE Hash Envelope.
+//
+// Reference: https://www.ietf.org/archive/id/draft-ietf-cose-hash-envelope-03.html
```

Comment:
Do we need to update the link to be consistent with others?
```suggestion
// Reference: https://www.ietf.org/archive/id/draft-ietf-cose-hash-envelope-04.html
```

Reply:
Nice catch. Updated.